packages/sdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@artblocks/sdk",
-  "version": "0.1.29",
```

Comment:
do we want the extra `-0`?

Reply:
Yeah, that's how yarn4 version command does prerelease versioning and didn't want to do a full release for now.

see https://yarnpkg.com/cli/version